app/controllers/ConfirmationController.scala

```diff
@@ -32,7 +34,8 @@ class ConfirmationController @Inject()(appConfig: FrontendAppConfig,
                                          ) extends FrontendController with I18nSupport {
 
   def onPageLoad: Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    Ok(confirmation(appConfig))
+    val data: Confirmation = request.userAnswers.get(ConfirmationPage).get
```

Comment:
Remove this `.get`
See how this is done in `ConfidentialInformationController`

Reply:
It's a different scenario to `ConfidentialInformationController` as the data is required. Maybe it could redirect if it isnt

Reply:
Sorted this